components/Heading/index.test.js

```diff
@@ -1,8 +1,10 @@
 import Heading from ".";
 import { render, screen } from "@testing-library/react";
 
-test("test", () => {
-	render(<Heading>🐠</Heading>);
-	const element = screen.getByText("🐠");
-	expect(element).toBeInTheDocument();
+test("renders the Heading component with an image and title", () => {
+	render(<Heading />);
+	const titleElement = screen.getByText("My Capstone Project:");
```

Comment:
Please consider the [query priority of React Testing Library](https://testing-library.com/docs/queries/about/#priority) and use `getByRole` here instead to make your test checking for a11y as well.